Library/Homebrew/cask/cmd/uninstall.rb

```diff
@@ -46,7 +46,7 @@ def self.uninstall_casks(*casks, binaries: nil, force: false, verbose: false)
           next if (versions = cask.versions).empty?
 
           puts <<~EOS
-            #{cask} #{versions.to_sentence} #{"is".pluralize(versions.count)} still installed.
+            #{cask} #{versions.to_sentence} #{versions.count == 1 ? "is" : "are"} still installed.
```

Comment:
it's possible that `versions.one?` makes sense in substitutions like these, but there are some edge cases i didn't want to risk